datumaro/cli/contexts/project/diff.py

```diff
@@ -243,11 +243,11 @@ def save_item_bbox_diff(self, item_a, item_b, diff):
         _, mispred, a_unmatched, b_unmatched = diff
 
         if 0 < len(a_unmatched) + len(b_unmatched) + len(mispred):
-            if not item_a.has_image or not item_a.image.has_data:
+            if not item_a.media:
```

Comment:
I think, we need to have a check somewhere in this class for media type and use `media_as()` for type deduction

Reply:
Added check for media type. But I didn't understand why we need to use `media_as() `there.